### PR TITLE
DOC-4793

### PR DESCRIPTION
**Pull request by jamesnocentini**
https://issues.couchbase.com/browse/DOC-4793

### CODE REVIEW

**Comment by jamesnocentini** [comment]
In the current form of the code snippet, `query.RemoveChangeListener(token);` is being called immediately after `query.Execute()` and this may be confusing to users who expect the code example to work as is.

@Sandychuang8 @borrrden is there a different syntax in C# to separate the live query setup and live query listener removal? I changed something here but I'm pretty sure that wouldn't compile.

**Comment by borrrden** [comment]
A `using` statement is just sugar for the following:

```c#
Query q;
try {
    q = <whatever>
    q.DoStuff();
} finally {
    q.Dispose();
}
```

So if you want to separate them just remove the using statement all together and be sure to call `Dispose()` later.

**Comment by jamesnocentini** [comment]
@borrrden ok thanks.

@Sandychuang8 Sorry I didn't notice that until the [previous PR](https://github.com/couchbase/docs-couchbase-lite/pull/70) was merged. Might it be possible to update it per Jim's suggestion? You can push a commit this PR. Thanks!

(in swift [live-query](https://github.com/couchbase/docs-couchbase-lite/blob/release/2.1/modules/ROOT/examples/swift/code-snippets/SampleCodeTest.swift#L226) and [stop-live-query](https://github.com/couchbase/docs-couchbase-lite/blob/release/2.1/modules/ROOT/examples/swift/code-snippets/SampleCodeTest.swift#L244))

**Comment by Sandychuang8** [comment]
@jamiltz It's a good practice to put query into a `using` statement. I don't think we should modify what we have.

File: modules/ROOT/examples/csharp/Program.cs [hunk 1]
```diff
@@ -306,30 +306,31 @@ private static void SelectAll()
...
+            var token = query.AddChangeListener((sender, args) =>
+            {
+                var allResult = args.Results.AllResults();
+                foreach (var result in allResult) {
+                    Console.WriteLine(result.Keys);
+                    /* Update UI */
+                }
+            });
+			
+			// Start live query.
```
Comment (jamesnocentini) [inline modules/ROOT/examples/csharp/Program.cs]:
```suggestion
            // Start live query.
```

File: modules/ROOT/examples/csharp/Program.cs [hunk 2]
```diff
@@ -306,30 +306,31 @@ private static void SelectAll()
             // end::query-select-all[]
 
             // tag::live-query[]
-            using (var query = QueryBuilder
+            var query = QueryBuilder
                 .Select(SelectResult.All())
-                .From(DataSource.Database(db))) {
+                .From(DataSource.Database(Db));
```
Comment (jamesnocentini) [inline modules/ROOT/examples/csharp/Program.cs]:
```suggestion
                .From(DataSource.Database(db));
```

**Review by borrrden — APPROVED** [review]

**Pull request merged**